docs/deploying/k8s.md

```diff
@@ -0,0 +1,98 @@
+# Setting up and using Kubernetes
```

Comment:
would state the scope of this, which is debugging what is going wrong with Kubernetes; ideally all that is needed should be to interact with the PR and the link added on the commit status